.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# data
+data/en_ner_bionlp13cg_md-0.5.1/
```

Comment:
Können wir diesen Ordner in einen subfolder von data verschieben? Am besten data/external/en_ner_biolon13cg_md-0.5.1/